lib/galaxy/managers/configuration.py

```diff
@@ -158,7 +157,6 @@ def _config_is_truthy(item, key, **context):
             'allow_user_creation': _defaults_to(False),  # schema default is True
             'use_remote_user': _defaults_to(None),  # schema default is False; or config.single_user
             'single_user': _config_is_truthy,
-            'enable_oidc': _use_config,
```

Comment:
`enable_oidc` seems to still be used in a number of places, removed by mistake?

Reply:
ah, yes!